UM/View/GL/Texture.py

```diff
@@ -32,12 +63,28 @@ def bind(self, texture_unit):
             if self._file_name != None:
                 self._image = QImage(self._file_name).mirrored()
             elif self._image is None: # No filename or image set.
-                self._image = QImage(1, 1, QImage.Format.Format_ARGB32)
+                self._image = QImage(self._fallback_width, self._fallback_height, QImage.Format.Format_ARGB32)
                 self._image.fill(0)
             self._qt_texture.setData(self._image)
-            self._qt_texture.setMinMagFilters(QOpenGLTexture.Filter.Linear, QOpenGLTexture.Filter.Linear)
-
+            self._qt_texture.setMinMagFilters(self._aa_filter, self._aa_filter)
+        self._performSubImageUpdates()
         self._qt_texture.bind(texture_unit)
+
+    def setSubImage(self, image: QImage, x: int, y: int) -> Optional[QImage]:
+        xrange = range(self._image.width())
+        yrange = range(self._image.height())
+        if (not (x in xrange and y in yrange)) or (not ((x + image.width()) in xrange and (y + image.height()) in yrange)):
```

Comment:
I think this statement becomes a bit more readable using the `not in` operator. This way some parenthesis can be removed.
```suggestion
        if (x not in xrange) or (y not in yrange) or ((x + image.width()) not  in xrange) or ((y + image.height()) not  in yrange):
```